sktime/forecasting/model_selection/_tune.py

```diff
@@ -53,10 +54,10 @@ def __init__(
         self.scoring = scoring
         self.verbose = verbose
         self.return_n_best_forecasters = return_n_best_forecasters
+        self.best_forecaster_ = forecaster
```

Comment:
I think `__init__` is not the right place for attributes. See also here: https://scikit-learn.org/stable/developers/develop.html#instantiation